src/mmw/js/src/modeling/gwlfe/quality/templates/table.html

```diff
@@ -14,22 +14,27 @@
     <tbody>
         {% for row in rows %}
             <tr>
-                {% for value in row %}
-                    {% if loop.index0 == 0 %}
-                        <td class="text-left">{{ value }}</td>
-                    {% else %}
-                        <td class="strong text-right">{{ value|round(2)|toLocaleString(2) }}</td>
+              {% set precision = default_precision %}
+              {% for value in row %}
+                {% if loop.index0 == 0 %}
```

Comment:
That wasn't intentional, I just went with the editor defaults and didn't check the formatting. I'll commit a fix before merging.